.gitignore

```diff
@@ -0,0 +1,2 @@
+# Ignore the generated binary from 'go build'
+/gospel
```

Comment:
I'd rather not have this here. It's not necessary.

Reply:
Uh, okay.  I don't follow, since `go build` in that directory will make the file and git will try to add it if you just add everything.  But sure, I've removed it and force-pushed to drop it.

I've added it to my local `.git/info/exclude` instead, so I won't make silly mistakes.  Sorry, was trying to be helpful.